homeassistant/components/shelly/sensor.py

```diff
@@ -56,6 +56,8 @@
 class BlockSensorDescription(BlockEntityDescription, SensorEntityDescription):
     """Class to describe a BLOCK sensor."""
 
+    suggested_display_precision: int | None = None
+
```

Comment:
```suggestion
```

This should be provided by the SensorEntityDescription already


https://github.com/home-assistant/core/blob/1759f58fc110b6c3ad81d859d6e5716182ad29d2/homeassistant/components/sensor/__init__.py#L149